modules/ztp-precaching-downloading-artifacts.adoc

```diff
@@ -278,13 +278,13 @@ kind: ImageSetConfiguration
 mirror:
   platform:
     channels:
-    - name: stable-4.11
-      minVersion: 4.11.5 <1>
-      maxVersion: 4.11.5
+    - name: stable-4.12
+      minVersion: 4.12.0 <1>
+      maxVersion: 4.12.0
   additionalImages:
     - name: quay.io/custom/repository
   operators:
-    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.11
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.12
```

Comment:
Remove 

             - name: 'release-2.5'
               minVersion: 2.5.4		               minVersion: 2.5.4
               maxVersion: 2.5.4		               maxVersion: 2.5.4

             - name: 'stable-2.0'
               minVersion: 2.0.4		               minVersion: 2.0.4
               maxVersion: 2.0.4		               maxVersion: 2.0.4

Reply:
The imageset for MCE and ACM would be like this:

    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.12
      packages:
        - name: multicluster-engine
          channels:
            - name: 'stable-2.1'
              minVersion: 2.1.4
              maxVersion: 2.1.4
        - name: advanced-cluster-management
          channels:
            - name: 'release-2.6'
              minVersion: 2.6.3
              maxVersion: 2.6.3

Reply:
There are also some changes to the operators. This is the imageset for 4.12.z

apiVersion: mirror.openshift.io/v1alpha2
kind: ImageSetConfiguration
mirror:
  platform:
    channels:
    - name: stable-4.12
      minVersion: 4.12.0
      maxVersion: 4.12.0
  additionalImages:
    - name: quay.io/custom/repository
  operators:
    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.12
      packages:
        - name: multicluster-engine
          channels:
            - name: 'stable-2.1'
              minVersion: 2.1.4
              maxVersion: 2.1.4
        - name: advanced-cluster-management
          channels:
            - name: 'release-2.6'
              minVersion: 2.6.3
              maxVersion: 2.6.3
        - name: local-storage-operator
          channels:
            - name: 'stable'
        - name: ptp-operator
          channels:
            - name: 'stable'
        - name: sriov-network-operator
          channels:
            - name: 'stable'
        - name: cluster-logging
          channels:
            - name: 'stable'
        - name: lvms-operator
          channels:
            - name: 'stable-4.12'
        - name: amq7-interconnect-operator
          channels:
            - name: '1.10.x'
        - name: bare-metal-event-relay
          channels:
            - name: 'stable'
    - catalog: registry.redhat.io/redhat/certified-operator-index:v4.12
      packages:
        - name: sriov-fec
          channels:
            - name: 'stable'